sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -31,6 +31,7 @@
 import com.google.common.collect.Iterators;
 import com.google.common.collect.Lists;
 import com.google.common.io.Closeables;
+
```

Comment:
Nit: you have a bunch of newlines added in-between imports that probably the IDE's "auto-indent" added, they are unnecessary.